osu.Game/Overlays/TabControlOverlayHeader.cs

```diff
@@ -90,7 +92,7 @@ private class OverlayHeaderTabItem : OverlayTabItem
                 public OverlayHeaderTabItem(T value)
                     : base(value)
                 {
-                    Text.Text = value.ToString().ToLower();
+                    Text.Text = ((value as Enum)?.GetDescription() ?? value.ToString()).ToLower();
```

Comment:
This should be (a) probably split to a method, since it's getting unreadable, and something like this even could go miles for readability:
```csharp

public string FormatHeaderText(object value)
{
    string baseText;

    switch (value)
    {
        case string stringValue:
            baseText = stringValue;
            break;
        case Enum enumValue:
            baseText = enumValue.GetDescription();
            break;
        default:
            baseText = enumValue.ToString();
            break;
    }

    return baseText.ToLower();
}
```
and I'd even go further to say (b) this needs to be made a global extension helper as the same sort of logic is already popping up in other places (see `HomeTitle`).

Reply:
Yeah, I thought to move it to some kind of a helper class as well, but wasn't sure

Reply:
May be it's a good idea to just move it to `TabControlOverlayHeader`. It will be used only once there